CHANGELOG.md

```diff
@@ -3,7 +3,96 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.38.0
+
+This release contains a number of fixes and enhancements.
+
+It contains one **backwards-incompatible change** to the JSON representation
+of metrics in **Status API** payloads, please see the section below.
+
+### Rich Metadata
+
+It is now possible to annotate Rego policies in a way that can be
+processed programmatically, using _Rich Metadata_.
+
+    # METADATA
+    # title: My rule
+    # description: A rule that determines if x is allowed.
+    # authors:
+    # - Jane Austin <jane@example.com>
+    allow {
+      ...
+    }
+
+The available keys are:
+
+- title
+- description
+- authors
+- organizations
+- related resources
```

Comment:
related_resources ?

Reply:
yeah well that's the key... but it's not... the meaning...? 🤔

Reply:
🤦 but the line above says "keys". Will fix.

Reply:
We explicitly say that we're listing keys up top, though 😉. The docs describe each key in more detail.